### PR TITLE
Ensuring log dir is only queries for selinux info if it exists

### PR DESCRIPTION
**Pull request by jpodivin**
I've noticed some uncaught errorrs in the cifmw-doc execution.[0] I'm going to try some minimal changes to address them.

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running

[0] https://review.rdoproject.org/zuul/build/fc109862c5814c1692300d79bd637742/console

### CODE REVIEW

File: ci/playbooks/collect-logs.yml [hunk 1]
```diff
@@ -17,9 +17,10 @@
             cmd: cp /tmp/report.html .
 
         - name: Check if we get ci-framework-data basedir
-          register: cifmw_state
```
Comment (cjeanner) [inline ci/playbooks/collect-logs.yml:20]: it's being used on line 27 (or 26).

File: ci/playbooks/collect-logs.yml [hunk 2]
```diff
@@ -17,13 +17,12 @@
...
-          ansible.builtin.stat:
+          ansible.builtin.file:
             path: "{{ ansible_user_dir }}/ci-framework-data"
+            state: directory
+            mode: "0755"
 
         - name: Collect ci-framework-data content of interest
-          when:
-            - cifmw_state.stat.exists | bool
```
[the remark this answers is not in the document]
Comment (jpodivin) [reply ci/playbooks/collect-logs.yml:26]: If you think this change doesn't make sense I'll drop it. My preference would be to have the directory by default, but I'll follow your suggestion.

**Comment by cjeanner** [comment]
/approve

We may iterate if needed - but this sounds better than previous iterations. Thanks Jiri!

**Review by raukadah — APPROVED** [review]
/approve
/lgtm

**Pull request merged**